go.mod

```diff
@@ -3,74 +3,27 @@ module github.com/coreos/prometheus-operator
 go 1.12
 
 require (
-	github.com/Azure/azure-storage-blob-go v0.0.0-20181022225951-5152f14ace1c // indirect
-	github.com/NYTimes/gziphandler v1.1.1 // indirect
-	github.com/StackExchange/wmi v0.0.0-20180725035823-b12b22c5341f // indirect
-	github.com/VividCortex/ewma v1.1.1 // indirect
 	github.com/ant31/crd-validation v0.0.0-20180702145049-30f8a35d0ac2
-	github.com/biogo/store v0.0.0-20160505134755-913427a1d5e8 // indirect
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/brancz/gojsontoyaml v0.0.0-20190425155809-e8bd32d46b3d
 	github.com/campoy/embedmd v1.0.0
-	github.com/cenk/backoff v2.0.0+incompatible // indirect
-	github.com/certifi/gocertifi v0.0.0-20180905225744-ee1a9a0726d2 // indirect
-	github.com/cockroachdb/apd v1.1.0 // indirect
-	github.com/cockroachdb/cmux v0.0.0-20170110192607-30d10be49292 // indirect
-	github.com/cockroachdb/cockroach v0.0.0-20170608034007-84bc9597164f // indirect
-	github.com/cockroachdb/cockroach-go v0.0.0-20181001143604-e0a95dfd547c // indirect
-	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
-	github.com/elastic/gosigar v0.9.0 // indirect
-	github.com/elazarl/go-bindata-assetfs v1.0.0 // indirect
-	github.com/facebookgo/clock v0.0.0-20150410010913-600d898af40a // indirect
-	github.com/fatih/structtag v1.0.0 // indirect
 	github.com/fortytw2/leaktest v1.3.0 // indirect
-	github.com/getsentry/raven-go v0.1.2 // indirect
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-ini/ini v1.21.1 // indirect
 	github.com/go-kit/kit v0.9.0
-	github.com/go-ole/go-ole v1.2.4 // indirect
 	github.com/go-openapi/spec v0.19.2
-	github.com/go-sql-driver/mysql v1.4.1 // indirect
 	github.com/gogo/protobuf v1.2.2-0.20190730201129-28a6bbf47e48
 	github.com/golang/protobuf v1.3.2
-	github.com/googleapis/gax-go v2.0.2+incompatible // indirect
-	github.com/gopherjs/gopherjs v0.0.0-20181103185306-d547d1d9531e // indirect
-	github.com/gregjones/httpcache v0.0.0-20180305231024-9cad4c3443a7 // indirect
-	github.com/grpc-ecosystem/go-grpc-middleware v1.0.0 // indirect
-	github.com/grpc-ecosystem/grpc-opentracing v0.0.0-20180507213350-8e809c8a8645 // indirect
-	github.com/hashicorp/consul v1.4.4 // indirect
 	github.com/hashicorp/go-version v1.1.0
 	github.com/improbable-eng/thanos v0.3.2
-	github.com/jackc/fake v0.0.0-20150926172116-812a484cc733 // indirect
-	github.com/jackc/pgx v3.2.0+incompatible // indirect
 	github.com/jsonnet-bundler/jsonnet-bundler v0.1.0
-	github.com/knz/strtime v0.0.0-20181018220328-af2256ee352c // indirect
 	github.com/kylelemons/godebug v0.0.0-20170820004349-d65d576e9348
-	github.com/leanovate/gopter v0.2.4 // indirect
```

Comment:
I ran `go mod tidy` 🤷‍♀

Reply:
Seems like this might be because I switched to 1.13 go version, as the CI which is on 1.12 fails. Installing older version and trying again.